rules.unified

```diff
@@ -55,7 +60,6 @@ KODI_CONFIG_OPTIONS ?= --host=$(DEB_HOST_GNU_TYPE) \
   --enable-nfs \
   --enable-libcec \
   --enable-afpclient \
-  --enable-airtunes \
```

Comment:
Please see latest. Reversed disabling of airtunes. The aim is to allow libguntls-dev. I apologize for the mistake.